libraries/eddystone-advertising-library/bower.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "eddystone-advertising-library",
+  "version": "0.3",
+  "homepage": "https://github.com/google/eddystone/tree/master/libraries/eddystone-advertising-library",
```

Comment:
Someone could also provide an implementation for Node on [bleno](https://github.com/sandeepmistry/bleno), so "javascript" is probably better than "web-js".